scss/bootstrap/overrides/style/_forms.scss

```diff
@@ -19,11 +18,18 @@ label {
     border-color: $brand-primary;
   }
 
-  &[disabled],
-  &[readonly],
-  fieldset[disabled] & {
+  &[disabled] {
     color: $crm-grayblue-darker;
   }
+
+  &[readonly] {
+    background-color: $crm-white;
+    color: $crm-gray-matte;
+  }
+}
+
+[disabled] .form-control {
```

Comment:
also, shouldn't this be 
```scss
.form-control {
  [disabled] & {
    // ...
  }
```

Reply:
Yes, when I removed the `fieldset` it allowed me to do so. I thought it didn't like the `selector &`.